admin/schema.php

```diff
@@ -770,4 +770,12 @@ function installer_db_now() {
 
 # Release marker: 1.3.0-rc.1
 
+$g_upgrade[202] = array( 'AlterColumnSQL', array( db_get_table( 'api_token' ), "
+	user_id					I		UNSIGNED NOTNULL DEFAULT '0',
+	name					C(128)	NOTNULL DEFAULT \" '' \",
```

Comment:
I'm not sure why you are adding a default.  It is by design that there is no default since this field was part of the initial definition of the table and we don't want any entries to be added without setting this field.  Hence, we shouldn't have a default value.